modules/snmp/README.md

```diff
@@ -6,219 +6,254 @@ sidebar_label: "SNMP"
+| charts.dimensions.divisor    |       1        | the value to divide the collected value, applied to convert it properly to units                                 |
+
+### Example: using SNMPv1/2
 
 In this example:
 
--   the SNMP device is `10.11.12.8`.
--   the SNMP version is `3` (defined under `options.version`)
--   we will update the values every 3 seconds (`update_every: 3` for the server `10.11.12.8`).
--   since we are using SNMPv3, we need to define authentication mechanisms under `user`
--   we define 1 chart `example` having 2 dimensions: `in` and `out`.
+- the SNMP device is `192.0.2.1`.
+- the SNMP version is `2`.
+- the SNMP community is `public`.
+- we will update the values every 10 seconds.
+- we define 2 charts `bandwidth_port1` and `bandwidth_port2`, each having 2 dimensions: `in` and `out`.
+
+> **SNMPv1**: just set `options.version` to 1.
 
 ```yaml
 jobs:
-  - name: local
-    update_every: 3
-    hostname: "10.11.12.8"
+  - name: switch
+    update_every: 10
+    hostname: "192.0.2.1"
+    community: public
     options:
-      port: 1161
-      version: 3
-      max_request_size: 60
-    user:
-      name: "username"
-      level: 3
-      auth_proto: 2
-      auth_key: "auth_key"
-      priv_proto: 2
-      priv_key: "priv_key"
+      version: 2
     charts:
-      - title: "example"
-        priority: 1
+      - id: "bandwidth_port1"
+        title: "Switch Bandwidth for port 1"
         units: "kilobits/s"
         type: "area"
-        family: "lan"
+        family: "ports"
         dimensions:
           - name: "in"
-            oid: ".1.3.6.1.2.1.2.2.1.10.2"
+            oid: "1.3.6.1.2.1.2.2.1.10.1"
             algorithm: "incremental"
             multiplier: 8
             divisor: 1000
           - name: "out"
-            oid: ".1.3.6.1.2.1.2.2.1.16.2"
+            oid: "1.3.6.1.2.1.2.2.1.16.1"
+            multiplier: -8
+            divisor: 1000
+      - id: "bandwidth_port2"
+        title: "Switch Bandwidth for port 2"
+        units: "kilobits/s"
+        type: "area"
+        family: "ports"
+        dimensions:
+          - name: "in"
+            oid: "1.3.6.1.2.1.2.2.1.10.2"
+            algorithm: "incremental"
+            multiplier: 8
+            divisor: 1000
+          - name: "out"
+            oid: "1.3.6.1.2.1.2.2.1.16.2"
             multiplier: -8
             divisor: 1000
 ```
 
-`update_every` is the update frequency for each server, in seconds.
-
-`family` sets the name of the submenu of the dashboard each chart will appear under.
+### Example: using SNMPv3
 
-`multiplier` and `divisor` are passed by the plugin to the Netdata daemon and are applied to the metric to convert it properly to `units`. 
+To use SNMPv3:
 
+- use `user` instead of `community`.
+- set `options.version` to 3.
 
-If many charts need to be defined using incremental OIDs, `multiply_range` can be used as in following example:
+The rest as in the SNMPv1/2 [example](#example-using-snmpv12).
```

Comment:
```suggestion
#### SNMP v3 sample

The rest of the configuration is the same as in the SNMPv1/2 [example](#example-using-snmpv12).
```
If the rest is the same, what does the example below illustrate?

Reply:
The security parameters needed for SNMPv3 (auth and privacy)

```yaml
    user:
      name: "username"
      level: "authPriv"
      auth_proto: "sha256"
      auth_key: "auth_protocol_passphrase"
      priv_proto: "aes256"
      priv_key: "priv_protocol_passphrase"
```